core/src/solvers/pipeline_planner.cpp

```diff
@@ -67,9 +67,18 @@ PipelinePlanner::PipelinePlanner() {
 	                    planning_pipeline::PlanningPipeline::MOTION_PLAN_REQUEST_TOPIC);
 }
 
-void PipelinePlanner::init(const core::RobotModelConstPtr& robot_model) {
-	planner_ = Task::createPlanner(robot_model);
+PipelinePlanner::PipelinePlanner(const planning_pipeline::PlanningPipelinePtr& planning_pipeline) : PipelinePlanner() {
+	planner_ = planning_pipeline;
```

Comment:
Please initialize planner_ using the constructor initializer list.
Also, could you add an overloaded constructor for passing a non-default namespace?

Reply:
C++ doesn't allow using member initialize list when using delegating constructor [see](https://stackoverflow.com/questions/12190051/member-initialization-while-using-delegated-constructor)

Reply:
I would say creating a new property `p.declare<std::string>("planner_ns", "move_group", "desc");` and using it in the `planner_ = Task::createPlanner(robot_model, properties().get<std::string>("planner_ns"));` would replace the need to create new constructor + new member variable, but the reason why I didn't push it is because of the segfault I'm getting

Reply:
Sorry I forget to answer the segfault problem, I'm looking into it now but still can't figure out why it's happening

Reply:
With multiple constructors I would declare the properties in a separate function `declareProperties()`, so we don't need to delegate. Also, regarding the segfault, did you specify "planner_ns" or leave it empty?

Reply:
Both segfault, but when I upload the parameters to different namespace and set the property to that one it would segfault in case of ompl and work normally with chomp

Reply:
Ok, so that seems like an unrelated issue. I'm fine with merging this as is then.